recipes/attune/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "attune" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "071e2ff296681cab3194da35646bff7b864ab981e5b650ea3d5be5379615869f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```

Would update like so to [match the example]( https://github.com/conda-forge/staged-recipes/blob/fdc1ab5f16288dea426d5c53d4fccb7f37cd5c8a/recipes/example/meta.yaml#L33 ). These arguments are handled by conda-build behind the scenes.

```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

---

thanks! should have read more carefully...